dbs/mongo/queries/addAnswer.js

```diff
@@ -0,0 +1,30 @@
+const {Answers, Answer_Photos} = require('../models.js');
+const {addAnswerPhoto} = require('./addPhotos.js')
+
+async function addAnswer(q_id, body, name, email, photos){
+  let findHighestA_ID = await Answers.find().sort({answer_id: -1}).limit(1);
+
+  let a_id = findHighestA_ID[0].answer_id + 1;
+
+  if (photos.length > 0) {
+    await Promise.all(photos.map(async (photo) => {
+      await addAnswerPhoto(a_id, photo)
```

Comment:
Is it necessary to use Promise.all when you are using async - await? And where those promises get resolved / rejected?